util/tinyjson.py

```diff
@@ -11,7 +11,7 @@
 def clsKey(cls):
     return cls.__qualname__
 
-def register(cls):
+def register(cls, tag=None):
```

Comment:
This enables setting the `__jsontag__` manually, instead of using the built in name generating algorithm. Needed if registering identically-named classes.